WordPress/Classes/ViewRelated/Post/PostEditor+Publish.swift

```diff
@@ -318,30 +319,31 @@ extension PostEditor where Self: UIViewController {
         postEditorStateContext.updated(isBeingPublished: true)
 
         uploadPost() { [weak self] uploadedPost, error in
-            guard let strongSelf = self else {
+            guard let self = self else {
                 return
             }
-            strongSelf.postEditorStateContext.updated(isBeingPublished: false)
+            self.postEditorStateContext.updated(isBeingPublished: false)
             SVProgressHUD.dismiss()
 
             let generator = UINotificationFeedbackGenerator()
             generator.prepare()
 
             if let error = error {
                 DDLogError("Error publishing post: \(error.localizedDescription)")
-
-                SVProgressHUD.showDismissibleError(withStatus: action.publishingErrorLabel)
+                SVProgressHUD.dismiss()
+                let model = PostNoticeViewModel(post: self.post)
+                ActionDispatcher.dispatch(NoticeAction.post(model.notice))
```

Comment:
⚠️ This `Notice` shows a _Retry_ button which changes the workflow. Should we consider using the previous message with no _Retry_ button instead? 

| Before | After |
|--------|-------|
|  ![2019-05-10 08 08 31 gifcask 2019-05-10 08_13_11](https://user-images.githubusercontent.com/198826/57533628-7a8e1380-72fb-11e9-8bf0-9a386e7d8ddc.gif) |     ![2019-05-10 07 48 40 2019-05-10 07_58_12](https://user-images.githubusercontent.com/198826/57533646-7f52c780-72fb-11e9-9d54-c1a2d4ce81c8.gif)|

Reply:
Thanks. I think you're right. There's no need for an update to the workflow here with the "retry"
I removed it and now showing informative notice.
![update_error_message](https://user-images.githubusercontent.com/1335657/57551575-3829fe00-731e-11e9-9c19-a8de004c9853.gif)